Src/WindowsServer/WindowsServer.Shared/AzureWebAppRoleEnvironmentTelemetryInitializer.cs

```diff
@@ -13,13 +13,13 @@
     /// </summary>
     public class AzureWebAppRoleEnvironmentTelemetryInitializer : ITelemetryInitializer
     {
-        /// <summary>Azure Web App name corresponding to the resource name.</summary>
-        private const string WebAppNameEnvironmentVariable = "WEBSITE_SITE_NAME";
-
         /// <summary>Azure Web App Hostname. This will include the deployment slot, but will be same across instances of same slot.</summary>
         private const string WebAppHostNameEnvironmentVariable = "WEBSITE_HOSTNAME";
 
-        private string roleInstanceName;
+        /// <summary>Predefined suffix for Azure Web App Hostname.</summary>
+        private const string WebAppSuffix = ".azurewebsites.net";
```

Comment:
Does this still work if they use a custom domain?
I'm not sure if the default Azure one still exists, or is at least usable, when users/companies use their own URL.
Assuming it is available, it may not be what they are expecting to see or search for in the portal/data.